ci/tests.py

```diff
@@ -223,12 +223,10 @@ class ScenarioSmokeConfigdriveIso9660Cdrom(BaseWindowsScenario):
     availability_zone = 'configdrive_iso9660_cdrom'
 
 
-@unittest.skipIf('static_network' not in AVAILABILITY_ZONES,
-                 'Needs special availability zone')
```

Comment:
put a note here that in the nova.conf for the kvm compute node, use_ipv6, force_config_drive need to be set to True